### PR TITLE
Release Notes NuGet 6.2

### PR DESCRIPTION
**Pull request by martinrrm**
I haven't cleaned the file yet, just created it

https://github.com/NuGet/NuGet.Client/compare/release-6.1.x...release-6.2.x

### CODE REVIEW

**Comment by JonDouglas** [comment]
@jebriede Can you help with small blurbs for #11486, #10752, #10751, #10748, #10749

@zivkan Any thoughts on a blurb for `nanoFramework`?

@jeffkl Are you okay with re-using our blog portions?

@nkolev92 Any idea for blurb on #11671?

**Comment by nkolev92** [comment]
I think `Add support for a dedicated environment variable providing the NuGetScratch path` is pretty good. 

I'd probably remove the feature prefixes from issues titles. 

We should also consider not including the sub features of larger feature in the main part.
Right now it's not immediately obvious what the most valuable part in `What's New` is.

Our tooling/labeling probably needs special considerations :)

File: docs/release-notes/NuGet-6.2.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+* Arrow keys in NuGet PM UI Sources editing doesn't change order of persistence - [#8315](https://github.com/NuGet/Home/issues/8315)
+
+* PackageReference ungracefully handles duplicate Runtime Identifiers in csproj PackageReference  - [#9290](https://github.com/NuGet/Home/issues/9290)
+
+* RestoreIgnoreFailedSources=true still gives warnings - [#9765](https://github.com/NuGet/Home/issues/9765)
+
+* Introduce a warning for null/empty version range (new or reuse NU1604) - [#9767](https://github.com/NuGet/Home/issues/9767)
+
+* NuGet restore in VS should report progress to allow the project-system to better control design time build scheduling.  - [#9782](https://github.com/NuGet/Home/issues/9782)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.2.md:72]: This should be a feature: https://github.com/NuGet/Home/issues/9782. 

We should include in the features with the title `Add IVsNuGetProjectUpdateEvents in Visual Studio, reporting of restore changes for PackageReference based projects.`  https://docs.microsoft.com/en-us/nuget/visual-studio-extensibility/nuget-api-in-visual-studio#ivsnugetprojectupdateevents-interface

File: docs/release-notes/NuGet-6.2.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+* [Responsiveness] Package Management UI can consume large number of threads all searching the disk (up to 316 threads), use cancellation token at subroutines - [#11599](https://github.com/NuGet/Home/issues/11599)
+
+* [Bug]: NU1004 in Visual Studio, but not command line (lock files in locked mode) - [#11639](https://github.com/NuGet/Home/issues/11639)
+
+* [Bug]: new warning for package source mappings doesn't pass a value for the resource string placeholder - [#11709](https://github.com/NuGet/Home/issues/11709)
+
+**StillOpens:**
+
+* GeneratePathProperty properties should end with a trailing slash - [#8871](https://github.com/NuGet/Home/issues/8871)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.2.md:100]: Still opens are meant to be deleted from the release notes/removed from the release.

**Review by nkolev92 — COMMENTED** [review]
Thanks for getting this going! 

A common thing when we use automation is that sometimes the description are not very user friendly. :)

Added some recommendations.

File: docs/release-notes/NuGet-6.2.md (hunk 1 above)
Comment (martinrrm) [reply docs/release-notes/NuGet-6.2.md]: Should the link to the documentation be added in the release notes?

File: docs/release-notes/NuGet-6.2.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+title: NuGet 6.2 Release Notes
+description: Release notes for NuGet 6.2 including new features, bug fixes, and DCRs.
+author: <GithubAlias>
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.2.md:4]: Remember to fill these out.

File: docs/release-notes/NuGet-6.2.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio <VSYear> with.NET Core workload
+
+## Summary: What's New in 6.2
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.2.md:20]: This looks great to me.

**Review by nkolev92 — APPROVED** [review]
LGTM.

**Comment by nkolev92** [comment]
Actually maybe we should do a `known issue` change for https://github.com/NuGet/Home/issues/11564. 

I think I'm supposed to write something for that. 

I'll try to write up it later today.

File: docs/release-notes/NuGet-6.2.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+ms.topic: conceptual
+---
+
+# NuGet 6.2 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.2.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.2](https://visualstudio.microsoft.com/downloads/) | [6.0.202](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
```
Comment (martinrrm) [inline docs/release-notes/NuGet-6.2.md:16]: Not sure if [6.0.202] is the correct version, I just grabbed the latest one

File: docs/release-notes/NuGet-6.2.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+* Add TFM for .NET nanoFramework - [#10800](https://github.com/NuGet/Home/issues/10800)
+
+* [Feature]: Require package source mapping when using CPM - [#11505](https://github.com/NuGet/Home/issues/11505)
+
+* [Feature]: Allow overriding a centrally defined package version - [#11516](https://github.com/NuGet/Home/issues/11516)
+
+* [Feature]: Add support for a dedicated environment variable providing the NuGetScratch path. - [#11671](https://github.com/NuGet/Home/issues/11671)
+
+* [Feature]: Add IVsNuGetProjectUpdateEvents in Visual Studio, reporting of restore changes for PackageReference based projects.  - [#9782](https://github.com/NuGet/Home/issues/9782)  - [See documentation](https://docs.microsoft.com/en-us/nuget/visual-studio-extensibility/nuget-api-in-visual-studio#ivsnugetprojectupdateevents-interface)
```
Comment (martinrrm) [inline docs/release-notes/NuGet-6.2.md:30]: Is `See documentation` use correct?

File: docs/release-notes/NuGet-6.2.md (hunk 5 above)
Comment (dominoFire) [reply docs/release-notes/NuGet-6.2.md:16]: Accroding to https://github.com/NuGet/Client.Engineering/issues/1560 ; NuGet 6.2 ships in dotnet SDK 6.0.3xx and later

Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md:16]: It'll be 6.0.300 actually.

File: docs/release-notes/NuGet-6.2.md (hunk 6 above)
Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md]: That's one way of doing it. It's ok imo.

Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md]: you need to use a relative link to the api, not an http link.

File: docs/release-notes/NuGet-6.2.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+* Read and store the transitive origins of a package while reading installed packages from assets file - [#10751](https://github.com/NuGet/Home/issues/10751)
+
+* Add caching of the transitive dependencies data pulled from the lockfile (assets file) - [#10752](https://github.com/NuGet/Home/issues/10752)
+
+* Surface the transitive packages and its transitive origins through the search layer - [#11486](https://github.com/NuGet/Home/issues/11486)
+
+* NuGet.exe list from local packages folder does not work with the AllVersion flag - [#4537](https://github.com/NuGet/Home/issues/4537)
+
+* Project A referencing package B via AssetTargetFallback, doesn't use that same AssetTargetFallback to pull B's dependency package C - [#5957](https://github.com/NuGet/Home/issues/5957)
```
Comment (nkolev92) [inline docs/release-notes/NuGet-6.2.md:62]: I wrote up a guide for the ATF change, maybe we can link it? https://github.com/NuGet/Samples/pull/53

In particular I want to: 

* Get the samples PR reviewed and merged.
* Add a comment linking that here. 


cc @JonDouglas 

Do you think we should add this to the notable changes list? 

Do we want to add a `Known issues` section or something like that? 

Maybe we have a better that mentions that this issue has consequences. 

Open to ideas.

Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md:62]: The link should be https://github.com/NuGet/Samples/tree/main/AssetTargetFallbackTransitiveDependencies.

Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md:62]: @dominoFire suggests that we can add `More info` and link to the example, in particular https://github.com/NuGet/Samples/tree/main/AssetTargetFallbackTransitiveDependencies.

Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md]: Pushed this. 

Should be complete now.

Comment (JonDouglas) [reply docs/release-notes/NuGet-6.2.md]: `known issues` best for anything that is known with the release and will addressed soon or has significant limitations when using today.

Comment (nkolev92) [reply docs/release-notes/NuGet-6.2.md]: This one is basically, you *might* have some weirdness, but there's no proposed changes right now.

See More Information, what we have right now, might work better for that. 

Feel free to tweak if you think it's necessary

File: docs/release-notes/NuGet-6.2.md (hunk 5 above)
Comment (dominoFire) [reply docs/release-notes/NuGet-6.2.md]: Changed to 6.0.300

**Comment by dominoFire** [comment]
Is there something left to do? Can I merge this PR?

**Review by JonDouglas — APPROVED** [review]
Do feel free to merge with what we have today. I will likely go back and update feature call outs and such later.

**Comment by nkolev92** [comment]
Can we get them merged to live?

**Pull request merged**